string[] pose_names
---
bool success
